cmd/validations/validations.go

```diff
@@ -0,0 +1,33 @@
+package validations
+
+import (
+	"os"
+
+	"github.com/openshift/moactl/pkg/aws"
+	"github.com/openshift/moactl/pkg/logging"
+	rprtr "github.com/openshift/moactl/pkg/reporter"
+	"github.com/spf13/cobra"
+)
+
+// Validations will validate if CF stack/users exist
+func Validations(cmd *cobra.Command, _ []string) {
+	reporter := rprtr.CreateReporterOrExit()
+	logger := logging.CreateLoggerOrExit(reporter)
+	// Create the AWS client:
+	client, err := aws.NewClient().
+		Logger(logger).
+		Region(aws.DefaultRegion).
+		Build()
+	if err != nil {
+		reporter.Errorf("Error creating AWS client: %v", err)
+		os.Exit(1)
+	}
+
+	reporter.Infof("Validating cloudformation stack is exist")
+	stackExist, err := client.CheckStackReadyOrNotExisting(aws.OsdCcsAdminStackName)
+	if !stackExist || err != nil {
+		reporter.Errorf("Cloudformation stack is not exist. Run `moactl init` first")
```

Comment:
message should be: "Cloudformation stack does not exist. Run `moactl init` first"